<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>battleViewerDialog</class>
 <widget class="QMainWindow" name="battleViewerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyWing - Space ship battles</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="ScenarioView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::NoDrag</enum>
      </property>
      <property name="resizeAnchor">
       <enum>QGraphicsView::AnchorUnderMouse</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>934</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew_game">
     <property name="title">
      <string>New game</string>
     </property>
     <addaction name="actionNew_empty_game"/>
     <addaction name="actionNew_basic_set_game"/>
    </widget>
    <addaction name="menuNew_game"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuLibraries">
    <property name="title">
     <string>Libraries</string>
    </property>
    <addaction name="actionPilots"/>
    <addaction name="actionShips"/>
    <addaction name="actionScenarios"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_basic_set"/>
    <addaction name="actionAdd_pilot"/>
   </widget>
   <widget class="QMenu" name="menuShow_range">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionRange_ruler"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuLibraries"/>
   <addaction name="menuShow_range"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowTitle">
    <string>Battle log</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTextEdit" name="logTextEdit">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSave_as_Pdf">
   <property name="text">
    <string>Export...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export image</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionActionAddPilot">
   <property name="text">
    <string>actionAddPilot</string>
   </property>
  </action>
  <action name="actionNew_basic_set_game">
   <property name="text">
    <string>New basic set game</string>
   </property>
  </action>
  <action name="actionPilots">
   <property name="text">
    <string>Pilots</string>
   </property>
  </action>
  <action name="actionShips">
   <property name="text">
    <string>Ships</string>
   </property>
  </action>
  <action name="actionScenarios">
   <property name="text">
    <string>Scenarios</string>
   </property>
  </action>
  <action name="actionRange_ruler">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Range ruler</string>
   </property>
  </action>
  <action name="actionAdd_basic_set">
   <property name="text">
    <string>Add basic set</string>
   </property>
  </action>
  <action name="actionNew_empty_game">
   <property name="text">
    <string>New empty game</string>
   </property>
  </action>
  <action name="actionAdd_pilot">
   <property name="text">
    <string>Add pilot</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScenarioView</class>
   <extends>QGraphicsView</extends>
   <header>Qt/scenarioView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExport</sender>
   <signal>triggered()</signal>
   <receiver>battleViewerDialog</receiver>
   <slot>export()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>battleViewerDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>battleViewerDialog</receiver>
   <slot>fileSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew_basic_set_game</sender>
   <signal>triggered()</signal>
   <receiver>battleViewerDialog</receiver>
   <slot>newBasicSetGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRange_ruler</sender>
   <signal>triggered()</signal>
   <receiver>battleViewerDialog</receiver>
   <slot>toggleShowRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdd_basic_set</sender>
   <signal>triggered()</signal>
   <receiver>battleViewerDialog</receiver>
   <slot>addBasicSet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew_empty_game</sender>
   <signal>triggered()</signal>
   <receiver>battleViewerDialog</receiver>
   <slot>newGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdd_pilot</sender>
   <signal>triggered()</signal>
   <receiver>battleViewerDialog</receiver>
   <slot>addPilot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>export()</slot>
  <slot>fileOpen()</slot>
  <slot>fileSave()</slot>
  <slot>addPilot()</slot>
  <slot>addBasicSet()</slot>
  <slot>toggleShowRange()</slot>
  <slot>newGame()</slot>
  <slot>newBasicSetGame()</slot>
 </slots>
</ui>
